code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -20,7 +20,6 @@
 	if(!current_area)
 		return
 	RegisterSignal(current_area, COMSIG_AREA_POWER_CHANGE, .proc/AreaPowerCheck)
-	AddElement(/datum/element/wall_mount)
```

Comment:
KYLERRRRR (also move this above the area check pls thxs

Reply:
its a duplicate of a line already doing it above it github just isnt showing it in the unexpanded view

Reply:
I hate old me